static/js/contribute.js

```diff
@@ -0,0 +1,42 @@
+//Make the DIV element draggagle:
+dragElement(document.getElementById(("contribute-box")));
+
+function dragElement(elmnt) {
+  var pos1 = 0, pos2 = 0, pos3 = 0, pos4 = 0;
+  if (document.getElementById(elmnt.id + "-tabs")) {
+    /* if present, the header is where you move the DIV from: */
+    document.getElementById(elmnt.id + "-tabs").onmousedown = dragMouseDown;
```

Comment:
Rather than setting `onmousedown` directly, use [`addEventListener`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) so this won't clobber any other code which happens to be listening for events and call [`removeEventListener`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener) when necessary.